types/google-maps-web/index.d.ts

```diff
@@ -0,0 +1,7699 @@
+// Type definitions for non-npm package Google Maps JavaScript API 3.42
+// Project: https://developers.google.com/maps/
+// Definitions by: Justin Poehnelt <https://github.com/jpoehnelt>
+//                 Alex Muramoto <https://github.com/amuramoto>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Generated by an automated process. DO NOT EDIT!
```

Comment:
Just double-checking: if there's a place for contributions, you should link to it here. But from our earlier discussion I don't think that's the case.